### ISSUE TITLE
BeaconActivity opens with error message

### ISSUE THREAD

**Issue by Ditscheridou** [helpscout/beacon-android-sdk-sample#92]
* Beacon version: 2.1.0 
* Android OS version: 9.0
* Device make/model: Xiaomi A3

**Steps to reproduce**
1. Following the initial tutorial on your site

**Actual outcome**
When the Helpscout activity opens the only outcome i see is "An unexpected error occurs, please try again later". I should mention that i am trying to open it from a fragment. 

I see this error in the logs: 
`    java.lang.IllegalStateException: Handler (android.os.Handler) {2e511d2} sending message to a Handler on a dead thread
        at android.os.MessageQueue.enqueueMessage(MessageQueue.java:546)
        at android.os.Handler.enqueueMessage(Handler.java:745)
        at android.os.Handler.sendMessageAtTime(Handler.java:697)
        at android.os.Handler.sendMessageDelayed(Handler.java:667)
        at android.os.Handler.post(Handler.java:395)
        at android.hardware.camera2.impl.CameraDeviceImpl$CameraHandlerExecutor.execute(CameraDeviceImpl.java:2372)
        at android.hardware.camera2.impl.CameraDeviceImpl.close(CameraDeviceImpl.java:1188)
        at com.sensetime.faceunlock.Camera2WrapperImpl.closeCamera(Camera2WrapperImpl.java:305)
        at com.sensetime.faceunlock.FaceService.stopUnlock(FaceService.java:1333)
        at com.sensetime.faceunlock.FaceService.access$1300(FaceService.java:70)
        at com.sensetime.faceunlock.FaceService$FaceServiceWrapper.cancelAuthentication(FaceService.java:959)
        at com.wos.android.face.IFaceService$Stub.onTransact(IFaceService.java:95)
        at android.os.Binder.execTransact(Binder.java:731)`

**Expected outcome**
the activity opens normally

**Screenshots or GIFs**
Any examples of the issue

**Comment by scottyab** [comment]
Hey @DitscheridouI curious as this stacktrace doesn't mention Beacon at all? as I've just mentioned in #91  it would be helpful for you to post the code you are using. 

Please can you confirm you are initialising the Beacon like this in something like your Application Object or Dependancy Injection:
```
Beacon beacon = new Beacon.Builder()
  .withBeaconId("YOUR BEACON ID")       
  .build();
```
And launching Beacon via `BeaconActivity.open(context);`?

Thanks

**Comment by Ditscheridou** [comment]
i can confirm that i am doing this. I know that this stacktrace doesnt mention Helpscout, but it only occures when i am trying to open the Beacon Activity.

**Comment by Ditscheridou** [comment]
okay nevermind this error error above is not related to this. But anyway im still receiving this error: 
![Screenshot_20200424-123055](https://user-images.githubusercontent.com/7279950/80203809-080b6600-8628-11ea-8bbe-44d7b326650a.png)

**Comment by aliripe** [comment]
Hi @Ditscheridou, Alicia here joining Scott 👋 

I'm sorry to see that you're running into issues with the SDK. As Scott mentioned, could you please provide the relevant snippets of code in which you're:
1. Setting up the SDK 
2. Starting the SDK from the `Fragment`

This will help us investigate and try to replicate the issue on our side so that you can start using the SDK as soon as possible!

**Comment by scottyab** [comment]
Hi @Ditscheridou we've just fixed the other issue you raised #91 I believe these we're linked. However, please do reopen this if you're still experiencing the error.
